src/components/Link/utils.ts

```diff
@@ -1,27 +1,9 @@
 import { px } from '../../utils';
-import { LinkVariants as UpdatedLinkVariants } from './types';
+import { LinkVariants } from './types';
 
-export const getLinkVariantClassName = (variant: keyof typeof UpdatedLinkVariants) => `${px}-link--${variant}`;
+export const getLinkVariantClassName = (variant: keyof typeof LinkVariants) => `${px}-link--${variant}`;
 
 export const isLinkExternal = (href?: string) =>
   !!href?.match(
     /(http[s]?:\/\/)(?!.*phillips\.com)([a-zA-Z0-9\-.]+)(:[0-9]{1,4})?([a-zA-Z0-9/\-._~:?#[\]@!$&'()*+,;=]*)/g,
   );
-
-// Leaving here for backwards compatibility. Can remove on next major release
```

Comment:
breaking semver here. Should we merge this and the Prefix change together and release v2?

Reply:
each one will make it's own version, but I will rev this one.